src/styles/blocks/stop-watch.scss

```
@@ -0,0 +1,66 @@
+.stop-watch {
+  position: relative;
+
+  width: 30vw;
+  height: 30vw;
+
+  border-radius: 50%;
+  background-image: url("../images/stopWatchBackground.jpg");
+  background-size: contain;
+
+  &__center {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    z-index: 4;
+    transform: translate(-50%, -50%);
+
+    width: 5%;
+    height: 5%;
+
+    border-radius: 50%;
+    background-color: black;
```

u use 50% quite a lot, so maybe it would be a good idea to create a variable (change it everywhere)

```suggestion
    top: variable;
    left: variable;
    z-index: 4;
    transform: translate(-variable, -variable);

    width: 5%;
    height: 5%;

    border-radius: 50%;
    background-color: black;
```

---

I don't think so.
This values are independent.
Position calculates from parrent positional block
Transform/translate calculates from self size

I am not sure but let's leave it up to the mentor

thank you for review)